setup/nextcloud.sh

```diff
@@ -285,47 +302,48 @@ if [ \( $? -ne 0 \) -a \( $? -ne 3 \) ]; then exit 1; fi
 
 # Set PHP FPM values to support large file uploads
 # (semicolon is the comment character in this file, hashes produce deprecation warnings)
-tools/editconf.py /etc/php/7.2/fpm/php.ini -c ';' \
-	upload_max_filesize=16G \
-	post_max_size=16G \
-	output_buffering=16384 \
-	memory_limit=512M \
-	max_execution_time=600 \
-	short_open_tag=On
+../tools/editconf.py /etc/php/7.2/fpm/php.ini -c ';' \
+        upload_max_filesize=16G \
+        post_max_size=16G \
+        output_buffering=16384 \
+        memory_limit=512M \
+        max_execution_time=600 \
+        short_open_tag=On
 
 # Set Nextcloud recommended opcache settings
-tools/editconf.py /etc/php/7.2/cli/conf.d/10-opcache.ini -c ';' \
-	opcache.enable=1 \
-	opcache.enable_cli=1 \
-	opcache.interned_strings_buffer=8 \
-	opcache.max_accelerated_files=10000 \
-	opcache.memory_consumption=128 \
-	opcache.save_comments=1 \
-	opcache.revalidate_freq=1
+../tools/editconf.py /etc/php/7.2/cli/conf.d/10-opcache.ini -c ';' \
+        opcache.enable=1 \
+opcache.enable_cli=1 \
+        opcache.interned_strings_buffer=8 \
+        opcache.max_accelerated_files=10000 \
+        opcache.memory_consumption=128 \
+        opcache.save_comments=1 \
+        opcache.revalidate_freq=1
+
+# Configure the path environment for php-fpm
+../tools/editconf.py /etc/php/7.2/fpm/pool.d/www.conf -c ';' \
+                env[PATH]=/usr/local/bin:/usr/bin:/bin
```

Comment:
This needs a better description for it's addition.

Reply:
ok, didn't make any new comments